controllers/celebrities.controllers.js

```diff
@@ -0,0 +1,31 @@
+const Celebrity = require("../models/celebrity.model")
```

Comment:
Tienes tu archivo con C mayuscula por lo que genera error, podrias cambiarlo igual a c para que la improtacion no se rompa
```suggestion
const Celebrity = require("../models/Celebrity.model")
```